tests/integration/it_employees_test.py

```diff
@@ -15,3 +15,28 @@ def test_insert_employee(db_session):
     db_session.commit()
     row = db_session.query(Employee).get(employee.id)
     assert row.username == "alice"
+
+
+def test_timestamp_mixin(db_session):
+    """Integration test for testing TimestampsMixin"""
+    before = datetime.utcnow()
+    employee = Employee(
+        first_name="John", last_name="Smith",
+        username="john", phone_number="123", birth_date=datetime.utcnow(),
+        registration_date=datetime.utcnow(), account_status="A",
+        user_status="Working", email="test@test.com", password="bla",
+        pin_code=1234
+    )
+    db_session.add(employee)
+    db_session.commit()
+    row = db_session.query(Employee).get(employee.id)
+    assert row.created_on > before
```

Comment:
@krzyk looks good, but this test checks 2 things: one is the ``createdon`` date and the other is the ``updatedon`` mechanism.

I would therefore separate the tests. Up to this line it is the ``createdon`` test and the code bellow it should be in a new test, since it's asserting ``updatedon`` (and the fact that ``createdon`` doesn't change, which is ok.